droid-configs.inc

```diff
@@ -177,6 +177,16 @@ Configure sailfish eg naturally landscape devices like mako
 
 ################################################################
 
+%package flashing
+Provides: on-device-flash-tools
```

Comment:
+1 for droid-config-flashing (droid denotes device related, configs says which package it might be [unless we externalise in future], and "flashing" is keeping it simple:)